superset/migrations/versions/2024-08-13_15-17_39549add7bfc_remove_sl_table_columns_table.py

```diff
@@ -26,17 +26,19 @@
 from alembic import op
 
 from superset.migrations.shared.constraints import drop_fks_for_table
+from superset.migrations.shared.utils import has_table
 
 # revision identifiers, used by Alembic.
 revision = "39549add7bfc"
 down_revision = "02f4f7811799"
 
 
 def upgrade():
-    connection = op.get_bind()
-    if connection.dialect.name != "sqlite":
-        drop_fks_for_table("sl_table_columns")
-    op.drop_table("sl_table_columns")
+    table_name = "sl_table_columns"
```

Comment:
Can you mode this outside the `upgrade` method (below `down_revision` variable) and reuse it on `downgrade`?

Reply:
Please do this for all changed migrations.

Reply:
done